ui/src/components/settings/sub_pages/licenses.rs

```diff
@@ -0,0 +1,31 @@
+use common::icons::outline::Shape as Icon;
+use dioxus::prelude::*;
+use kit::elements::button::Button;
+use kit::elements::Appearance;
+use warp::logging::tracing::log;
+
+use crate::components::settings::SettingSection;
+
+#[allow(non_snake_case)]
+pub fn Licenses(cx: Scope) -> Element {
+    log::trace!("Licenses settings page rendered.");
+
+    cx.render(rsx!(
+        div {
+            id: "settings-licenses",
+            aria_label: "settings-licenses",
+            SettingSection {
+                section_label: "heroicons".into(),
+                section_description: "We have expanded upon the heroicons library we offer any additional icons under the same license as the original author.".into(),
```

Comment:
This should probably get moved into translation so we can use something like `get_local_text("settings-licenses.description")`

Reply:
See comment in discord, I don't think it is worth translating this since the end license they will be clicking is in English only anyways.